tracer/test/Datadog.Trace.ClrProfiler.IntegrationTests/DataStreamsMonitoringManualApiTest.cs

```diff
@@ -0,0 +1,49 @@
+// <copyright file="DataStreamsMonitoringManualApiTest.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+using System.Threading.Tasks;
+using Datadog.Trace.Configuration;
+using Datadog.Trace.TestHelpers;
+using Datadog.Trace.TestHelpers.DataStreamsMonitoring;
+using FluentAssertions;
+using VerifyXunit;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Datadog.Trace.ClrProfiler.IntegrationTests;
+
+[UsesVerify]
+public class DataStreamsMonitoringManualApiTest : TestHelper
+{
+    public DataStreamsMonitoringManualApiTest(ITestOutputHelper output)
+        : base("DataStreams.ManualAPI", output)
+    {
+        SetServiceVersion("1.0.0");
+    }
+
+    [SkippableFact]
+    [Trait("Category", "EndToEnd")]
+    public async Task ContextPropagation()
+    {
+        SetEnvironmentVariable(ConfigurationKeys.DataStreamsMonitoring.Enabled, "1");
+
+        using var telemetry = this.ConfigureTelemetry();
+        using var agent = EnvironmentHelper.GetMockAgent(useTelemetry: true);
+        using var processResult = await RunSampleAndWaitForExit(agent);
+
+        var spans = agent.WaitForSpans(count: 2);
+        spans.Should().HaveCount(expected: 2);
+        spans[index: 1].TraceId.Should().Be(spans[index: 0].TraceId); // trace context propagation
```

Comment:
It's very unusual to add the `index` arg to indexers 🙂 
```suggestion
        spans[1].TraceId.Should().Be(spans[0].TraceId); // trace context propagation
```

Reply:
ah yes I think it was added by my "reformat on save", I probably need to check how it's setup 👹